test_projects/example/bundlex.exs

```diff
@@ -34,9 +23,10 @@ defmodule Example.BundlexProject do
         sources: ["foo_nif.c"],
         interface: [:nif],
         os_deps: [
-          {PrecompiledFFmpeg, [:libswscale, :libavcodec]},
-          :libpng
-        ]
+          {get_ffmpeg_path(), ["libswscale", "libavcodec"]},
```

Comment:
I thought about this implicit pkg-config as a default, and maybe we could accept a list of 'providers', which could be either `{:precompiled, url}` or `:pkg_config` or `nil`. So in this case we'd have

```suggestion
          {[get_ffmpeg_path(), :pkg_config], ["libswscale", "libavcodec"]},
```

And `get_ffmpeg_path()` would return `{:precompiled, url}` or nil. This way is more verbose but less implicit and the user can decide whether to take the precompiled or OS package first ;) WDYT?

Reply:
done ;)